src/util/log.cc

```diff
@@ -269,11 +278,11 @@ logger::do_log(log_level level, log_writer& writer) {
       return;
     }
     static array_map<sstring, 20> level_map = {
-            { int(log_level::debug), "DEBUG" },
-            { int(log_level::info),  "INFO "  },
-            { int(log_level::trace), "TRACE" },
-            { int(log_level::warn),  "WARN "  },
-            { int(log_level::error), "ERROR" },
+            { int(log_level::debug), COLORED(fmt::color::white,  "DEBUG") },
+            { int(log_level::info),  COLORED(fmt::color::wheat,  "INFO ")  },
```

Comment:
isn't it better to leave INFO level as the default color? i.e. specify no one.

Reply:
i am using black now to simplify the code. but IMHO, is `INFO` always used to represent more uncommon messages compared to DEBUG? so it deserves a more visible color? i mean, probably TRACE should use `black` instead?